audible.py

```diff
@@ -24,6 +24,8 @@ def __init__(self):
             'match_chapters': True,
             'source_weight': 0.0,
             'include_narrator_in_artists': True,
+            'remove_series_reference_in_title': False,
+            'remove_series_reference_in_subtitle': False,
```

Comment:
Since there is already president set by `include_narrator_in_artists`, could this flag be inverted so that it defaults to true rather than false? I.e, `remove_series_reference_in_title` -> `include_series_in_title`? Likewise for `remove_series_reference_in_subtitle`.

Reply:
I see where you are coming from, but I disagree. The plugin isn't including/adding series info into those fields. It is removing what is provided by audnexus.